statistics/cmsketch.go

```diff
@@ -520,67 +468,103 @@ func (c *CMSketch) Copy() *CMSketch {
 	return &CMSketch{count: c.count, width: c.width, depth: c.depth, table: tbl, defaultValue: c.defaultValue}
 }
 
+// AppendTopN appends a topn into the TopN struct.
+func (c *TopN) AppendTopN(data []byte, count uint64) {
```

Comment:
How about making it unexposed, which may avoid some unnecessary modifies by mistake?

Reply:
statistics/handle will also call `AppendTopN`. It cannot be done currently.